ddl/ddl_api.go

```diff
@@ -2675,6 +2675,9 @@ func (d *ddl) AlterTable(ctx context.Context, sctx sessionctx.Context, ident ast
 		switch spec.Tp {
 		case ast.AlterTableAddColumns:
 			if len(spec.NewColumns) != 1 {
+				if !sctx.GetSessionVars().EnableChangeMultiSchema {
```

Comment:
Could we put this after https://github.com/pingcap/tidb/pull/29828/files#diff-b7c54f7dad7257cbe4fa04b8d76c9bed694e2e2e7c5599f45eb2172d8d4ff8a7R2648-R2671 and extract it into a function to check if it should have an `errRunMultiSchemaChanges`? I think it's a little bit more unified.

Reply:
Done, PTAL @zimulala